modules/analysis/lsh_test.py

```
@@ -1,83 +1,72 @@
-        display_column = args["target_column"]  # '누적시청시간' 등
-        target_column = target_column_map.get(display_column)  # 실제 DB 컬럼명
+class CohortAnalysis(Resource):
+    @cohort_ns.response(200, "Analysis successful")
+    @cohort_ns.response(400, "Bad request")
+    @cohort_ns.response(500, "Server error")
+    def post(self):
+        """Cohort analysis API (POST, returns CSV)"""
+        data = request.get_json()
+        info_db_no = data.get("info_db_no")
+        user_info = data.get("user_info")
+        user_sub_info = data.get("user_sub_info")
```

Ruff (0.11.9)

Local variable `user_sub_info` is assigned to but never used

Remove assignment to unused variable `user_sub_info`

(F841)